posts-router.js

```diff
@@ -0,0 +1,93 @@
+const express = require("express")
+const db = require("./data/db.js");
+
+const router = express.Router()
+
+router.post("/", (req, res) => {
+    const { title, contents, created_at, updated_at } = req.body;
+    if (!title || !contents) {
+      res.status(400).json({
+        errorMessage: "Please provide title and contents for the post."
+      });
+    }
+    db.insert({
```

Comment:
This didn't work for me during my review, but changing it to `db.insert(req.body)` did.